Add interfaces/svg-animations.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/svg-animations.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389

stephenmcgruer commented on interfaces/svg-animations.idl:
```diff
@@ -0,0 +1,60 @@
+
+interface TimeEvent : Event {
+
+  readonly attribute @@ unknown term "AbstractView" view;
```
'@@ unknown term "AbstractView"'

^ well thats unusual. It's actually in the IDL at https://svgwg.org/specs/animations/#InterfaceTimeEvent (scroll down a bit from the anchor), so I think we need to patch the spec

stephenmcgruer replied:
And turns out foolip already did. https://github.com/w3c/svgwg/issues/775

stephenmcgruer replied:
The blocking SVG issue has been fixed, yay!

stephenmcgruer commented:
This will need an equivalent idlharness test.

autofoolip committed:
Add interfaces/svg-animations.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/svg-animations.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389
